### PR TITLE
[Backport][ipa-4-6] Fix expected file permissions for ghost files

### PR DESCRIPTION
**Pull request by flo-renaud**
Manual backport of PR #3144 to ipa-4-6 branch. Since python2 packages are also shipped in this branch, the test also calls rpm -V on the python2 pkgs.
The other difference with the main branch is the use of ipaplatform.NAME instead of osinfo.id since osinfo.id was introduced in the master and ipa-4-7 branches only.

### CODE REVIEW

File: freeipa.spec.in [hunk 1]
```diff
@@ -1670,19 +1670,19 @@ fi
...
-%ghost %config(noreplace) %{_sysconfdir}/ipa/nssdb/cert9.db
-%ghost %config(noreplace) %{_sysconfdir}/ipa/nssdb/key4.db
-%ghost %config(noreplace) %{_sysconfdir}/ipa/nssdb/pkcs11.txt
-%ghost %config(noreplace) %{_sysconfdir}/ipa/nssdb/pwdfile.txt
-%ghost %config(noreplace) %{_sysconfdir}/pki/ca-trust/source/ipa.p11-kit
+%ghost %attr(644,root,root) %config(noreplace) %{_sysconfdir}/ipa/nssdb/cert9.db
+%ghost %attr(644,root,root) %config(noreplace) %{_sysconfdir}/ipa/nssdb/key4.db
+%ghost %attr(644,root,root) %config(noreplace) %{_sysconfdir}/ipa/nssdb/pkcs11.txt
+%ghost %attr(600,root,root) %config(noreplace) %{_sysconfdir}/ipa/nssdb/pwdfile.txt
+%ghost %attr(644,root,root) %config(noreplace) %{_sysconfdir}/pki/ca-trust/source/ipa.p11-kit
```
Comment (rcritten) [inline freeipa.spec.in]: A little surprised we have sqlite support for this as the default in RHEL 7 AFAIR is still dbm. It doesn't hurt to have it here I suppose.

Comment (flo-renaud) [reply freeipa.spec.in]: Yes, you're right, in ipa-4-6 branch the default is still dbm. It doesn't seem to create any issue to declare the sqlite files in the spec file, though. I checked on a RHEL 7 system and on a fedora 27, and rpm -V output is the same.

Comment (tiran) [reply freeipa.spec.in]: IIRC I added the sqlite variants when we moved to a Fedora version with NSS SQLite default. I agree with Flo, it doesn't hurt to keep them.

**Comment by flo-renaud** [comment]
ipa-4-6:

* b7afb147722b74237a2ce03b1fb20e9094f8246e Fix expected file permissions for ghost files
* c7712c17f3e92a29c4ca55a29fe210abbc2adc58 ipatests: add integration test checking the files mode

**Pull request closed without merging**